ReadMe.md

```diff
@@ -33,6 +33,22 @@ You can register an object using one of two methods:
   It's [lifetime management](#lifetime-management) is `singletonPerContainer`.
   It can be resolved using the `identifier` or as part of the [group](#resolve-groups) `mySimilarObjects`. 
   
+* `container.registerFactory(identifier, resolver)`.
+
+  This registers an instance factory using the given string `identifier`. 
+  This is mainly aimed to use with strong typed languages like Typescript.
```

Comment:
I don't think we should promte this as a first port of call pattern for TypeScript. Registering instance like this is a workaround for some missing refelection features in the language of which other solutions could be built.